### DISCUSSION TITLE
Override instant loading

### DISCUSSION THREAD

**Discussion by patrick-5546** [squidfunk/mkdocs-material#5925] [Q&A] [answered]
Hi, I am using [mkdocs-drawio-file](https://github.com/onixpro/mkdocs-drawio-file) to insert a drawio diagram. However, it doesn't load when instant loading is enabled until the page is refreshed. Is there a way to disable instant loading just for the page that has the diagram? Or force it to refresh? Thanks

**Comment by squidfunk** [comment, marked as answer]
This has been asked multiple times. Check our [table sort documentation](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables) to see an example how to integrate with external libraries. It's actually quite simple – you just need to hook into `document$.subscribe`.
